spec/controllers/bids_controller_spec.rb

```diff
@@ -155,7 +155,7 @@
           expect(flash[:bid]).to eq("success")
           expect(response).to redirect_to("/auctions/#{auction.id}")
 
-          bid = auction.bids.order('created_at DESC').first
+          bid = auction.bids.last
```

Comment:
now or later? default ordering is by creation date and I am generally against default scopes / ordering so don't imagine it is likely to change, but it definitely could!